### PR TITLE
OHRM5X-1644 : Add notes for language strings

### PR DESCRIPTION
**Pull request by SharukaNP**

### CODE REVIEW

File: installer/Migration/V5_2_0/lang-string/notesForUnitIds.yaml [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+    - { group: performance, note: 'Rate the employee based on the KPI with the assigned Min rate', unitId: min_rate }
+    - { group: performance, note: 'Rate the employee based on the KPI with the assigned Min and Max Rate', unitId: max_rate }
+    - { group: performance, note: 'When mapping KPI for job titles, you can reserve one KPI as the default one', unitId: is_default }
+    - { group: performance, note: 'Rate the employee based on the KPI with the assigned Min rate', unitId: minimum_rating }
+    - { group: performance, note: 'Maximum rating that can be given for a particular KPI', unitId: maximum_rating }
+    - { group: performance, note: 'If you select “Make Default Scale” the defined minimum and maximum rating will be pre-populated for all KPIs added in the future.', unitId: make_default_scale }
+    - { group: performance, note: "Once added, they can be modified.Added and Modified dates are captured automatically.\r\n          Modified date log entries added to the tracker captured automatically.\n        ", unitId: modified_date }
```
Comment (RajithaKumara) [inline installer/Migration/V5_2_0/lang-string/notesForUnitIds.yaml:669]:
```suggestion
    - { group: performance, note: "Once added, they can be modified. Added and Modified dates are captured automatically. Modified date log entries added to the tracker captured automatically.", unitId: modified_date }
```

File: installer/Migration/V5_2_0/lang-string/notesForUnitIds.yaml [hunk 2]
```diff
@@ -0,0 +1,689 @@
+notesForUnitIds:
```
Comment (RajithaKumara) [inline installer/Migration/V5_2_0/lang-string/notesForUnitIds.yaml:1]:
```suggestion
notes:
```

File: installer/Migration/V5_2_0/Migration.php [hunk 3]
```diff
@@ -244,4 +246,23 @@ private function cleanLDAPAddonData(): void
             ->setParameter('moduleName', 'ldapAuthentication')
             ->executeQuery();
     }
+
+    private function insertLangStringNotes(): void
+    {
+        $filepath = $filepath = __DIR__ . '/lang-string/notesForUnitIds.yaml';
```
Comment (RajithaKumara) [inline installer/Migration/V5_2_0/Migration.php:252]:
```suggestion
        $filepath = $filepath = __DIR__ . '/lang-string/notes.yaml';
```

File: installer/Migration/V5_2_0/Migration.php [hunk 4]
```diff
@@ -244,4 +246,23 @@ private function cleanLDAPAddonData(): void
...
+        $yml = Yaml::parseFile($filepath);
+        $langStrings = array_shift($yml);
```
Comment (RajithaKumara) [inline installer/Migration/V5_2_0/Migration.php:254]:
```suggestion
        $langStrings = $yml['notes'];
```
:thinking:

**Review by RajithaKumara — CHANGES_REQUESTED** [review]
Good job :tada:

File: installer/Migration/V5_2_0/Migration.php (hunk 4 above)
Comment (SharukaNP) [reply installer/Migration/V5_2_0/Migration.php:254]: This notation won't work.

**Comment by SharukaNP** [comment]
Checked for special characters and removed them

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**